### PR TITLE
Update naming convention

### PR DESCRIPTION
**Pull request by JamesOsborn-SE**
Update naming convention to fit the current C# conventions

### CODE REVIEW

File: exercises/concept/elons-toys/.docs/introduction.md [hunk 1]
```diff
@@ -21,16 +21,16 @@ var myCar = new Car();
 var yourCar = new Car();
 ```
 
-Fields have a type and a name (defined in camelCase) and can be defined anywhere in a class (defined in PascalCase):
+Fields have a type (defined in camelCase) and can be defined anywhere in a class. Public fields are defined in PascalCase and private fields are defined with camelCase prefixed with an underscore `_`:
```
Comment (ErikSchierboom) [inline exercises/concept/elons-toys/.docs/introduction.md:24]: Tiny suggestion:

```suggestion
Fields have a type and can be defined anywhere in a class.
Public fields are defined in PascalCase and private fields are defined in camelCase and prefixed with an underscore `_`:
```

**Review by ErikSchierboom — COMMENTED** [review]
Lovely! One small suggestion.

File: exercises/concept/elons-toys/.docs/introduction.md (hunk 1 above)
Comment (JamesOsborn-SE) [reply exercises/concept/elons-toys/.docs/introduction.md]: I like it

**Comment by JamesOsborn-SE** [comment]
> 

Updated. I didn't know there was so much duplication or is there a generator I missed? 
Just an aside I really like the site keep up the good work!

**Review by ErikSchierboom — APPROVED** [review]
Neat!

**Comment by ErikSchierboom** [comment]
> Updated. I didn't know there was so much duplication or is there a generator I missed?

Yes, there is a generator that can generate the exercise's introduction.md file from the concept's introduction.md.  See https://exercism.org/docs/building/configlet/generating-documents

**Pull request merged**